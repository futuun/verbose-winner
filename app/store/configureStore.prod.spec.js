import configureStore from './configureStore.prod'

test('configureStore: initialize store', () => {
  expect(configureStore()).toMatchSnapshot()
})
