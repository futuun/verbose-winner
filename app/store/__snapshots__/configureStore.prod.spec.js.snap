exports[`test configureStore: initialize store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
